judo-ui-react/src/main/resources/actor/src/components/dialog/RangeDialog.tsx.hbs

```diff
@@ -386,16 +380,27 @@ export const RangeDialog = <T extends JudoStored<T>, U extends QueryCustomizer<T
               filters={filters}
               filterOptions={filterOptions}
               columns={columns}
-              onFiltersUpdated={ (newFilters: Filter[]) => {
-                setFilters(newFilters);
-                setPage(0);
-
-                setQueryCustomizer((prevQueryCustomizer) => {
-                  return {
-                    ...prevQueryCustomizer,
-                    ...mapAllFiltersToQueryCustomizerProperties(newFilters),
-                  };
-                });
+              onFilterByCell={ (filter: Filter) => {
+                  // LOFASZ
```

Comment:
Lofasz? What that mean? :D

Reply:
It is a necessary thing to create small horses.